### ISSUE TITLE
Rounded corners for prompts?

### ISSUE THREAD

**Issue by gyrocade** [Powerlevel9k/powerlevel9k#1378]
Are rounded (ellipsoid) ends possible for prompts?

**Comment by romkatv** [comment]
With Powerlevel9k you can specify custom heads (ends that point inwards) but not tails (ends that point outwards). There are half-circle glyphs in Nerd Fonts but I don't know if there are half ellipsoid glyphs with non-zero eccentricity anywhere. If you really want to, it's not too difficult to draw them by yourself.

**Comment by gyrocade** [comment]
Can you point me to some documentation for custom drawing the tails?  I want to do a Star Trek: TNG style prompt.

**Comment by romkatv** [comment]
> Can you point me to some documentation for custom drawing the tails? I want to do a Star Trek: TNG style prompt.

Star Trek: TNG console has rounded outward edges. Powerlevel9k doesn't support this.

By the way, the theme you want to use is literally "powerlevel9k", right? It's not some other theme that has a similar name? This subtle difference has significant implications on the answer to your question.

**Comment by gyrocade** [comment]
Yes, that's the ZSH theme I'm using with the nerd fonts set.  Sounds like you're implying another theme can do what I want to accomplish...

**Comment by romkatv** [comment]
> Sounds like you're implying another theme can do what I want to accomplish...

I wouldn't dare.

**Comment by gyrocade** [comment]
Thank you for your responses.
